flink-connectors/flink-connector-kinesis/src/main/java/org/apache/flink/streaming/connectors/kinesis/internals/ShardConsumer.java

```diff
@@ -96,6 +99,15 @@ protected ShardConsumer(KinesisDataFetcher<T> fetcherRef,
 							SequenceNumber lastSequenceNum,
 							KinesisProxyInterface kinesis) {
 		this.fetcherRef = checkNotNull(fetcherRef);
+		MetricGroup kinesisMetricGroup = fetcherRef.getRuntimeContext()
+			.getMetricGroup()
+			.addGroup("Kinesis")
```

Comment:
Do we really need this group? The metric is already bounded to the current subtask, which should provide enough context that it is Kinesis-related since we're the Kinesis consumer, no?

Reply:
Would be best if @zentol also comments on this.